Makefile

```diff
@@ -47,7 +47,7 @@ endif
 domserver: domserver-configure paths.mk config
 judgehost: judgehost-configure paths.mk config
 docs: paths.mk config
-install-domserver: domserver composer-dump-autoload domserver-create-dirs
+install-domserver: composer-dump-autoload domserver-create-dirs
```

Comment:
We should not remove this dependency of `install-domserver` on `domserver`. https://www.gnu.org/prep/standards/html_node/Standard-Targets.html shows that it is standard that an install target also compiles anything required. As also discussed in https://stackoverflow.com/questions/70901364/should-make-install-depend-on-compilation, I think that if you want/need to run `make install` as root, then you should make sure to first run `make build` (aka `make domserver`) as yourself or accept that files are owned by root.

I guess this doesn't work because `copy-bundle-assets` then gets executed again during `domserver-install`, but that's something that should be fixed by making it a real dependency (not a "phony target"). That way, if it was already created during `make domserver`, it won't get recreated during `make domserver-install` with root ownership.